mm2src/mm2_main/src/lp_swap/taker_swap_v2.rs

```diff
@@ -111,6 +111,14 @@ pub enum TakerSwapEvent {
         maker_payment: TransactionIdentifier,
         negotiation_data: StoredNegotiationData,
     },
+    /// Sent taker payment and taker payment spend preimage generation and sending were skipped
```

Comment:
I am not sure if it's just me but I didn't get what this suppose to mean. It's seems like the plain format of the `TakerPaymentSentAndPreimageSendingSkipped` type.

Reply:
Doc comment clarifies what preimage was skipped (taker payment spend preimage)

Reply:
Maybe "Taker payment is sent and preimage of 'taker payment spend' is skipped"?

Reply:
Agree, done https://github.com/KomodoPlatform/komodo-defi-framework/pull/2359/commits/2bc305b3d379fbc7ed4674c61b3158a249457e2b